packages/static-wado-scp/package.json

```diff
@@ -53,7 +53,12 @@
   },
   "dependencies": {
     "@ohif/static-wado-creator": "^0.5.1",
+    "@ohif/static-wado-plugins": "^0.5.1",
     "@ohif/static-wado-util": "^0.5.1",
+    "config-point": "^0.3.9",
     "dcmjs-dimse": "^0.0.14"
+  },
+  "devDependencies": {
+    "must": "^0.13.4"
```

Comment:
must is already added on root project, there is no need to add it again

Reply:
lint says otherwise, and is an error not to have it.  I prefer to have common dependencies at the top level, but the tools don't all agree on it.